docs/credentials/get-credential-status.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Get a Credential’s Status
+intro: Sometimes you may want to verify if a credential currently has the required permissions and access in a given provider. Get the Credential Status to validate if a credential is still active and has the necessary permissions.
```

Comment:
Could you use the article name in the intro?

I believe that helps the user fixate the article's goal

```suggestion
intro: Sometimes you may want to verify if a credential currently has the required permissions and access in a given provider. Get a Credential's Status to validate if a credential is still active and has the necessary permissions.
```